modules/rules/templates/rules/audit_logging_rules.yaml

```diff
@@ -0,0 +1,62 @@
+# Copyright 2018 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# rules:
```

Comment:
Does this file need to be included if it's completely commented out?

Reply:
This is a copy of the default Forseti rules (https://github.com/GoogleCloudPlatform/forseti-security/blob/dev/rules/audit_logging_rules.yaml); @ocervell @hshin-g could you two chime in on this?

Reply:
Since this is a rules file, it does need to be included.

Reply:
@hshin-g with regards to the commented out sections of the file, should we leave that as-is or should we delete those sections entirely? I believe that @aaron-lane's point was that if we're installing files we shouldn't ship files that entirely commented out. What's your take on that? Do you prefer that we ship the exact copies of what you have upstream or are you alright with us dropping commented out sections and blank files?

Reply:
Yes.